### PR TITLE
Update skill model unit tests

### LINKED ISSUE #531: Write unit tests for the Skill model
Write unit tests for the [`Skill` Model](https://github.com/code-corps/code-corps-ember/blob/develop/app/models/skill.js). There is an [existing unit test file](https://github.com/code-corps/code-corps-ember/blob/develop/tests/unit/models/skill-test.js), but it could use more coverage (computed properties & relationship declarations). 

## References
- Progress on: #98 
- [Model testing guide](https://guides.emberjs.com/v2.8.0/testing/testing-models/)
- [Contributing Guidelines](https://github.com/code-corps/code-corps-ember/blob/develop/.github/CONTRIBUTING.md)
- Join our [Slack](https://codecorps.slack.com)

### PR DESCRIPTION
**Pull request by kumarmj**
fixes #531

### CODE REVIEW

File: tests/unit/models/skill-test.js [hunk 1]
```diff
@@ -10,3 +10,14 @@ test('it exists', function(assert) {
...
+
+test('it has all of its attributes', function(assert) {
+  assert.expect(3);
+
+  const skill = this.subject();
+  const attributes = Object.keys(skill.toJSON());
+
+  assert.ok(attributes.includes('description'), 'should have description attribute');
+  assert.ok(attributes.includes('title'), 'should have title attribute');
+  assert.ok(attributes.includes('matched'), 'should have matched attribute');
```
Comment (WenInCode) [inline tests/unit/models/skill-test.js]: can you move this above the `title` attribute check (alpha ordering)

Comment (JoshSmith) [inline tests/unit/models/skill-test.js]: Once we merge @filipecrosk's we'll have to come back through and add his new `assert.hasAttributes`, too. Will make writing these tests much easier.

Comment (WenInCode) [inline tests/unit/models/skill-test.js]: @JoshSmith ya, I realize that too. I figure I'll just make a Pr once that is in to fix up all the already written tests.

Comment (JoshSmith) [inline tests/unit/models/skill-test.js]: @WenInCode maybe add an issue and mark as blocked.

**Comment by kumarmj** [comment]
Silly me, corrected alpha ordering, great idea `assert.hasAttributes`

**Comment by JoshSmith** [comment]
Blocked by #566. Will be good once that's merged.

**Comment by JoshSmith** [comment]
@kumarmj now you can use `hasAttributes`.

**Comment by JoshSmith** [comment]
🎉 thanks @kumarmj 🙌

**Pull request merged**